README.md

```diff
@@ -27,6 +28,38 @@ Rich IntelliSense (completions) for `Dockerfile` and `docker-compose.yml` files:
 
 ![IntelliSense for DockerFiles](images/intelliSense.gif)
 
+## Debugging (Preview)
+
+To debug a .NET Core web application running in a Linux Docker container, add a Docker .NET Core launch configuration:
+
+1. Switch to the debugging tab.
+1. Select `Add configuration...`
+1. Select `Docker: Launch .NET Core (Preview)`
+1. Set a breakpoint.
+1. Start debugging.
+
+> Mac OS X users: before debugging, add `/usr/local/share/dotnet/sdk/NuGetFallbackFolder` as a shared folder in your Docker preferences.
```

Comment:
A picture is worth a thousands words

Reply:
Added a thousand words (i.e. one picture).